packages/react/src/layouts/index.tsx

```diff
@@ -7,13 +7,15 @@ export type LayoutProps = {
   meta?: any
   noIndex?: boolean
   className?: string
+  showNavigation?: boolean
 }
 
 const Layout: React.FC<LayoutProps> = ({
   children,
   className,
   meta,
   noIndex,
+  showNavigation = true,
```

Comment:
would rather use a "slot" or some other pattern but I was getting tripped up on types so this is a "punt" that should be reconsidered.